src/cs_transmog.cpp

```diff
@@ -26,33 +26,33 @@ using namespace Acore::ChatCommands;
 class transmog_commandscript : public CommandScript
 {
 public:
-    transmog_commandscript() : CommandScript("transmog_commandscript") { }
+    transmog_commandscript() : CommandScript("transmog_commandscript") {}
 
     ChatCommandTable GetCommands() const override
     {
         static ChatCommandTable addCollectionTable =
-        {
-            { "set", HandleAddTransmogItemSet,    SEC_MODERATOR, Console::No },
-            { "",    HandleAddTransmogItem,       SEC_MODERATOR, Console::No },
-        };
+            {
+                {"set", HandleAddTransmogItemSet, SEC_MODERATOR, Console::Yes},
+                {"", HandleAddTransmogItem, SEC_MODERATOR, Console::Yes},
+            };
 
         static ChatCommandTable transmogTable =
-        {
-            { "add", addCollectionTable },
-            { "",    HandleDisableTransMogVisual, SEC_PLAYER,    Console::No },
-        };
+            {
+                {"add", addCollectionTable},
+                {"", HandleDisableTransMogVisual, SEC_PLAYER, Console::No},
+            };
 
         static ChatCommandTable commandTable =
-        {
-            { "transmog", transmogTable },
-        };
+            {
+                {"transmog", transmogTable},
+            };
 
         return commandTable;
     }
 
-    static bool HandleDisableTransMogVisual(ChatHandler* handler, bool hide)
+    static bool HandleDisableTransMogVisual(ChatHandler *handler, bool hide)
     {
-        Player* player = handler->GetPlayer();
+        Player *player = handler->GetPlayer();
```

Comment:
those codestyle changes don't comply, are they autogenerated?

Reply:
yes, when I save the file it automatically make them in this way 😓

Reply:
Same with the lines above. 

Could just turn that off or change the settings for formatting :P

Reply:
style fixed